tests/shell/tests.py

```diff
@@ -10,6 +10,13 @@
 
 class ShellCommandTestCase(SimpleTestCase):
 
+    SCRIPT_32183 = """if True:  # Ignore indentation
```

Comment:
Please don't use a ticket numbers in tests, maybe: `script_with_inline_function`.